st2-ask/notification_rule.yaml

```diff
@@ -0,0 +1,33 @@
+---
+name: "notify_slack_execution_change_managers"
+pack: "st2"
+description: Watches for an execution status of "pending" for a certain action ref
+enabled: true
+
+trigger:
+  type: core.st2.generic.actiontrigger
+
+criteria:
+  trigger.action_ref:
+      type: equals
+      pattern: st2.ask
+  trigger.status:
+      type: equals
+      pattern: pending
+
+  # In order to control who to notify, the "tag" parameter for the st2.ask action would be
+  # used to indicate metadata about this specific instance (would be set in the workflow).
+  # Since it's in the parameters, we can get a handle on it here in the rule and use it to
+  # inform our decision on who to notify.
+  #
+  # This is a very static, simple example - and would require one rule for each notification "group",
+  # or type of st2.ask action. See notification_rule_workflow.yaml for a more dynamic example
+  trigger.parameters.tag:
+      type: equals
+      pattern: managers
+
+action:
+  ref: slack.post_message
+  parameters:
+    channel: "#managers"
```

Comment:
For sure that's an option. It's a tradeoff on where the logic is placed. Option #1 is having to maintain multiple notification rules but as a result, no workflow is needed, and Option #2 is create one rule but also a notification workflow. My thought was that we'd just provide both suggestions since this is outside the core functionality of `st2.ask` anyways - just showing the user a few options that might work for them